portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/models/calendar/Property.java

```diff
@@ -0,0 +1,58 @@
+package org.dataportabilityproject.types.transfer.models.calendar;
+
+// This interface could be used by other models.  If so, we should move it out and make it more
+// broadly useful.
+interface Property<T> {
+  T getValue();
+}
+
+class RuleDay implements Property<RecurringEvent.Day> {
+  private RecurringEvent.Day day;
```

Comment:
final here and for the other implementation classes below

Reply:
Done.